review_practice/react-app/src/components/DatePicker/index.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+
+import 'react-dates/initialize';
+import 'react-dates/lib/css/_datepicker.css';
+import { DayPickerRangeController } from 'react-dates';
+
+import moment from 'moment';
+import 'moment/min/locales';
+
+import * as Style from './style';
+
+export default ({ requestToServer, datePicked, deleteButtonHandle, localeLanguage }) => {
+    moment.locale(localeLanguage);
```

Comment:
```suggestion
    useEffect(() => {
      moment.locale(localeLanguage);
    }, [localeLanguage]);
```

렌더 영역은 매번 실행되기 때문에 특정 함수를 실행하거나, 무언가를 세팅하기에는 올바르지 않은 위치입니다. effect에 넣어줘서 특정 디펜더시가 변경될때만 실행할 수 있도록 해주세요.

또한 `moment.locale()` 은 ui component가 아니라 최상위의 컴포넌트 위치에서 설정하는게 좀 더 옳지 않을까? 생각이 듭니다. 다국어를 고려하기 위한 컨텍스트에 설치하는게 가장 베스트일 것 같아요